x-pack/solutions/security/plugins/security_solution/public/asset_inventory/pages/all_assets.tsx

```diff
@@ -374,7 +374,7 @@ const AllAssets = ({
         loading={loadingState === DataLoadingState.loading}
       />
       <EuiPageTemplate.Section>
-        <EuiTitle size="l">
+        <EuiTitle size="l" data-test-subj="all_assets_title">
```

Comment:
Super nit: we typically use camelCase or kebab-case for this prop.

Reply:
Thanks, that's good to know!